services/src/test/java/org/fao/geonet/api/records/extent/MetadataExtentApiTest.java

```diff
@@ -154,7 +167,10 @@ public void lastModifiedModified() throws Exception {
             .andExpect(content().contentType(API_PNG_EXPECTED_ENCODING))
             .andReturn().getResponse().getContentAsByteArray();
 
-        assertEquals("b02baec6d92832ecd5653db78093a427", DigestUtils.md5DigestAsHex(reponseBuffer));
+        assertImage(
+            REFERENCE+"lastModifiedModified.png",
+            reponseBuffer,
+            37);
```

Comment:
@josegar74 this is an example of a tolerance that I guessed at to account for platform differences.

If you want you can reduce this to zero and we can get an actual difference between linux and macOS.